src/menus/PluginMenus.cpp

```diff
@@ -32,6 +32,383 @@
 // private helper classes and functions
 namespace {
 
+using MenuList = std::vector<std::pair<TranslatableString, std::vector<TranslatableString>>>;
+
+const MenuList& PreferredEffectsMenu()
+{
+   static MenuList items =
+   {
+      {
+         XO("Volume and compression"),
+         {
+            XO("Amplify"),
+            XO("Compressor"),
+            XO("Limiter"),
+            XO("Normalize"),
+            XO("Loudness Normalization"),
+            XO("Auto Duck")
+         }
+      },
+      {
+         XO("Fading"),
+         {
+            XO("Fade In"),
+            XO("Fade Out"),
+            XO("Studio Fade Out"),
+            XO("Adjustable Fade"),
+            XO("Crossfade Clips"),
+            XO("Crossfade Tracks")
+         }
+      },
+      {
+         XO("Pitch and Tempo"),
+         {
+            XO("Change Pitch"),
+            XO("Change Speed"),
+            XO("Change Tempo"),
+            XO("Paulstretch"),
+            XO("Sliding Stretch"),
+            XO("Reverse")
+         }
+      },
+      {
+         XO("EQ and Filters"),
+         {
+            XO("Bass and Treble"),
+            XO("Graphic EQ"),
+            XO("Filter Curve EQ"),
+            XO("High-Pass Filter"),
+            XO("Low-Pass Filter"),
+            XO("Notch Filter")
+         }
+      },
+      {
+         XO("Noise Removal and Repair"),
+         {
+            XO("Click Removal"),
+            XO("Noise Reduction"),
+            XO("Noise Gate"),
+            XO("Repair"),
+            XO("Clip Fix")
+         }
+      },
+      {
+         XO("Delay and Reverb"),
+         {
+            XO("Echo"),
+            XO("Reverb"),
+            XO("Delay")
+         }
+      },
+      {
+         XO("Distortion and Modulation"),
+         {
+            XO("Tremolo"),
+            XO("Distortion"),
+            XO("Wahwah"),
+            XO("Phaser"),
+            XO("Vocoder")
+         }
+      },
+      {
+         XO("Special"),
+         {
+            XO("Repeat"),
+            XO("Invert"),
```

Comment:
rename the `Invert` effect to `Invert Polarity`, please.

Reply:
Why?

Reply:
If it is agreed to make that name change, the correct change in Invert.cpp is:

```
const ComponentInterfaceSymbol EffectInvert::Symbol
{ L"Invert", XO("Invert Polarity") };
```

because in case anyone uses the Invert effect in a macro, this will avoid breaking that macro when they migrate to a newer version of Audacity.  Only the localized name of the effect will be changed, not the English identifier that macro programmers use.

Reply:
> Polarity



> rename the `Invert` effect to `Invert Polarity`, please.

As Paul wrote: "why?". What other kinds of waveform inversion are there? Yes there is such a thing as "spectral inversion" (which Audacity does not have), but if that's the distinction that you want to make then I don't think that "invert polarity" does it as well as "invert waveform".

Regardless of which wording is decided, please avoid changing the command name unless absolutely necessary as doing so will break custom macros that use the command. As Paul wrote: "Only the localized name of the effect" should be changed.